docker-compose.yaml

```diff
@@ -15,3 +15,12 @@ services:
   test:
     <<: *devbox
     command: docker/run_tests.sh --format-code
+
+  # generate and serve the project documentation locally
+  mkdocs:
```

Comment:
👍 

I know you mentioned wanting to generate a `/_dev` subdirectory for feature branch pushes, but I think viewing the content locally is much easier - this should be more than fine as a replacement imo

Reply:
`_dev/` is nice because it doesn't require the person reviewing the code to pull down the branch to look at the rendered content. But the benefit isn't enough to block this work from going forward.